server_health_check-rails.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "rails", ">= 3.0", "< 6.0"
-  spec.add_runtime_dependency "server_health_check", "~> 1.0"
+  spec.add_runtime_dependency "rails", "~> 5.0"
```

Comment:
This PR uses ruby 2.3 syntax.  I want to be able to release a rails-5 compatible gem that can leverage that syntax for convenience (`Module.prepend`).  This does not imply that we should not also have a rails-3 or rails-4 compatible gem.  The question is how to avoid developing on separate branches that just continue to diverge.